Policies/Kubernetes/forbidden-sysctl-profiles/rego/src.rego

```diff
@@ -0,0 +1,20 @@
+package k8sazureforbiddensysctls
+
+violation[{"msg": msg, "details": {}}] {
+    sysctl := input.review.object.spec.securityContext.sysctls[_].name
+    forbiddin_sysctl(sysctl)
+    msg := sprintf("The sysctl %v is not allowed, pod: %v. Forbidden sysctls: %v", [sysctl, input.review.object.metadata.name, input.parameters.forbiddenSysctls])
+}
+
+# * may be used to forbid all sysctls
+forbiddin_sysctl(sysctl) {
+    input.parameters.forbiddenSysctls[_] == "*"
+}
+
+forbiddin_sysctl(sysctl) {
+    input.parameters.forbiddenSysctls[_] == sysctl
+}
+
+forbiddin_sysctl(sysctl) {
+    startswith(sysctl, trim(input.parameters.forbiddenSysctls[_], "*"))
```

Comment:
Can you do a regex match here instead of startswith? As we do not validate the input parameter values, if user enters text like "kernel*forced" then the policy will be doing a wrong match.